mteb/tasks/Clustering/eng/ClusTrecCovidP2P.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+import itertools
+
+from datasets import Dataset, DatasetDict
+
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.AbsTaskClusteringFast import (
+    AbsTaskClusteringFast,
+    check_label_distribution,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ClusTrecCovidP2PFast(AbsTaskClusteringFast):
```

Comment:
```suggestion
class ClusTrecCovidP2PFast(AbsTaskClusteringFast, MultilingualTask):
```
Ahh for that you will need to add this one here. (in the v2 this will no longer be needed)

Reply:
pushed and edit PR